pvsite_datamodel/write/user_and_site.py

```diff
@@ -90,6 +99,9 @@ def create_site(
     if region in [None, ""]:
         region = "uk"
 
+    if asset_type not in SiteAssetType.__members__:
+        asset_type = SiteAssetType.pv.name
```

Comment:
or should we raise an error here?

Reply:
and could we add a test to check this behavour

Reply:
Yes to both!